source/testing_govuk_pay/index.html.md.erb

```diff
@@ -5,18 +5,25 @@ weight: 90
 
 # Testing GOV.UK Pay
 
-You will receive a test account for testing in addition to your live credentials.
+When you sign up for GOV.UK Pay, you will receive a test account in addition
+to your live credentials.
 
-When testing, you’ll need to ensure:
+When you test, you should make sure that you’ve tested with [mock card
+numbers](/#mock-card-numbers-for-testing-purposes) to simulate both successful
+and unsuccessful transactions. You must never use real card numbers for
+testing purposes, because this breaks PCI rules.
```

Comment:
i'm not totally sure this is true (about PCI), although it's probably good practice; the more important thing is that it just won't work. you have to use test cards with a test account.